release_notes/osd_3_2_release_notes.adoc

```diff
@@ -0,0 +1,60 @@
+= OpenShift Dedicated 3.2 Release Notes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+https://www.openshift.com/dedicated/[OpenShift Dedicated] is a new offering
+based on OpenShift Enterprise that provides a hosted OpenShift 3 environment to
+run the containers powering your applications.
+
+With OpenShift Dedicated, your cluster is isolated from other customers,
+providing security and management by the OpenShift Operations Team.
```

Comment:
I think "providing security" is an invalid concept.  Certainly it would make the lawyers salivate...

Suggest: "With OSD, your cluster is isolated from other customers and managed by the OpenShift Operations Team.  This increases security and performance stability."